soc/xtensa/intel_adsp/ace/Kconfig.series

```diff
@@ -6,6 +6,8 @@ config SOC_SERIES_INTEL_ACE
 	select SOC_FAMILY_INTEL_ADSP
 	select XTENSA
 	select XTENSA_HAL if ! XTENSA_TOOLCHAIN_HAS_HIFI
+	select XTENSA_ARCH_HAS_HIFI3 if XTENSA_TOOLCHAIN_HAS_HIFI
+	select XTENSA_ARCH_HAS_HIFI4 if XTENSA_TOOLCHAIN_HAS_HIFI
```

Comment:
there is no difference for HIFI3 and HIFI4, can we keep one only for ACE platform?